### PR TITLE
Use authors taxonomy and list all posts on a draft user's page

### PR DESCRIPTION
**Pull request by mgisbers**
This PR reworks how the author is used in this theme:

- taxonomy for authors gets activated in exampleSite's configuration
- exampleSite's blog entries use 'authors' instead auf 'author'
- exampleSite's blog entries have single, double and no author for testing
- correct linking to authors page

With this PR user already using 'author' entries in their pages/blog need change them from 'author' to 'authors' and the value from string ("John Doe") to list (["John Doe"]).

### CODE REVIEW

File: layouts/authors/list.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        {{ partial "breadcrumbs.html" . }}
+
+        <div id="content">
+            <div class="container">
+                <div class="row">
+                    <!-- *** LEFT COLUMN *** -->
+
+                    <div class="col-md-9" id="blog-listing-medium">
+
+                        <section class="post">
```
Comment (ryanfox1985) [inline layouts/authors/list.html:21]: perhaps is better to use the same structure as posts:

https://github.com/devcows/hugo-universal-theme/blob/master/layouts/_default/list.html#L19-L85

So we will have the same behaviour like tags or categories. In the future perhaps we can add support to author bios.

File: layouts/_default/list.html [hunk 2]
```diff
@@ -37,8 +37,12 @@
                                     <h2><a href="{{ .Permalink }}">{{ .Title }}</a></h2>
                                     <div class="clearfix">
                                         <p class="author-category">
-                                          {{ if isset .Params "author" }}
-                                          {{ i18n "authorBy" }} <a href="#">{{ .Params.author }}</a>
+                                          {{ if isset .Params "authors" }}
```
Comment (ryanfox1985) [inline layouts/_default/list.html:40]: there is a space before the comma:

Screenshot 2021-08-03 at 16 18 59

**Review by ryanfox1985 — COMMENTED** [review]
hey, now it looks good to me, I saw a couple of silly things:

File: layouts/_default/list.html (hunk 2 above)
Comment (mgisbers) [reply layouts/_default/list.html:40]: Fixed that.

File: layouts/authors/list.html (hunk 1 above)
Comment (mgisbers) [reply layouts/authors/list.html]: That makes it easier to implement ;-)

**Review by ryanfox1985 — APPROVED** [review]
LGTM

**Pull request merged**